src/js/controller/controller.js

```diff
@@ -482,7 +497,7 @@ Object.assign(Controller.prototype, {
             return 'external';
         }
 
-        function _autoStart() {
+        function _autoStart(autostartReason) {
```

Comment:
Would be nicer to name the argument `reason`.

Reply:
Wanted to, but it already exists in an inner scope.

Reply:
That refers to the `reason` in the `catch` block within `_autoStart`, which can be refactored to:
```
if (!checkAutoStartCancelable.cancelled()) {
    const code = getPlayAttemptFailedErrorCode(error);
    _this.trigger(AUTOSTART_NOT_ALLOWED, {
        reason: error.reason,
        code,
        error
    });
}
```

Reply:
Cool. Done.